scripts/build/docker/run-command.sh

```diff
@@ -82,6 +82,8 @@ fi
 # The `-t` and TERM setup is needed to display coloured output.
 docker run -t \
   --env "TERM=xterm-256color" \
+  --env "AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID" \
```

Comment:
It's worth noting that, as it is, I see no reason that I couldn't hijack your keys and publish whatever I wanted to your S3 repo. Even if the environment vars are encrypted, I could still open a PR that does `echo $AWS_ACCES_KEY_ID` and your secrets are spilled. In fact, it might automatically show up in the build log as commands are run...